latex/interni/doc_interna/norme_di_progetto/res/sections/01_content.tex

```diff
@@ -144,10 +167,12 @@ \subsubsection{Branching}
     \end{itemize}
 \textit{documentation}: 
 I documenti presenti in documentation sono stati approvati dal Responsabile di progetto o almeno verificati dai verificatori.\\
+Se si devono effettuare delle modifiche a branches protetti GitHub permette la creazione di un ramo di appoggio per la risoluzione dei conflitti. Successivamente, se non ci sono conflitti, verrà effettuato il merge e verrà eliminato il branch di appoggio.\\ 
```

Comment:
Il branch di appoggio non viene usato per modificare il branch protetto..
serve per integrare delle modifiche da il branch protetto ad uno libero.
Per modificare un branch protetto l'unico modo é una pull request.
Avevamo parlato di branch di appoggio per passare il file .gitignore da documentario a norme di progetto senza modificare documentation.
Infatti una pull aggiorna entrambi i rami che vengono mergiati. Al momento della conferma di una pull tra branch protetto e uno libero GitHub chiede se si vuole creare un branch di appoggio proprio per non modificare quello protetto.

Reply:
Ho ripensato a questo passaggio ed é meglio se questo passaggio viene fatto in locale per evitare errori involontari.

Per integrare delle modifiche da un branch protetto ad uno libero si utilizza un branch d’appoggio creato in locale partendo dall'ultimo commit di documentation e facendone il merge con il branch che necessita delle integrazioni. In seguito il branch di appoggio verrà eliminato.